components/kyma-operator/README.md

```diff
@@ -2,60 +2,65 @@
 
-- Docker
+The controller extends the api-server with the following CustomResources:
+- `installation.installer.kyma-project.io/v1alpha1`
 
-## Development
+## Prerequisites
 
-Before each commit, use the [`Makefile`](./Makefile) script to test your changes:
-  ```bash
-  make verify
-  ```
+- recent version of Go language with support for modules (e.g: 1.12.6)
+- make
+- kubectl
+- access to K8s environment: minikube or a remote K8s cluster
 
-## Rebuild Kubernetes client libraries
+## Details
 
-After introducing changes to the `apis/installer/v1alpha1` package, you have to regenerate client libraries by running:
-  ```
-  sh ./hack/update-codegen.sh
-  ```
+### Use command-line flags
 
-## Build a Docker image
+| Name | Required | Description | Default value |
+|------|----------|-------------|-----------------|
+| **kubeconfig** | no | Path to a kubeconfig file | Taken from the `KUBECONFIG` env parameter|
+| **kymaDir** | no | Directory holding helm releases, within the image/pod | `/kyma` |
+| **backoffIntervals** | no | Number of seconds to wait before subsequent retries | `10,20,40,60,80` |
+| **overrideLogFile** | no | Log File to Print Installation overrides to. | `/dev/stdout` |
+| **overrideLogFormat** | no | Installation Override Log format (Accepted values: text or json) | `text` |
+| **helmMaxHistory**  | no | Max number of releases returned by Helm release history query | `10` |
+| **helmDriver** | no | Helm backed storage drivers. [Options and details](https://helm.sh/docs/helm/helm/#helm) | `secrets` |
+| **helmDebugMode** | no | Enable/Disable Helm client output in the Kyma Operator logs | `false` |
 
-Use the [`Makefile`](./Makefile) script to build a Docker image of the Kyma Operator:
-  ```
-  export DOCKER_PUSH_REPOSITORY={DOCKER_REPOSITORY}
-  make build-image
-  ```
+### Supported Environmental Variables
 
-## Details
+| Name | Default | Description |
+| ---- | ------- | ----------- |
+| **INST_NAMESPACE** | `default` | Namespace in which the Installation CR is located |
+| **INST_RESOURCE** | `kyma-installation` | Name of the Installation custom resource |
+| **INST_FINALIZER** | `finalizer.installer.kyma-project.io` | Name of the finalizer |
+| **OVERRIDES_NAMESPACE** | `kyma-installer` | Namespace in which the Installer overrides are located |
 
-### Upgrade Kyma
+## Custom Resource
 
-The upgrade procedure relies heavily on Helm. When you trigger the upgrade, Helm performs `helm upgrade` on Helm releases that exist in the cluster and are defined in the Kyma version you're upgrading to. If a Helm release is defined in the Kyma version you're upgrading to but is not present in the cluster when you trigger the upgrade, Helm performs `helm install` and installs such a release.
+The `installation.installer.kyma-project.io` CustomResourceDefinition (CRD) is a detailed description of the kind of data and the format the Kyma Operator Controller listens for. To get the up-to-date CRD and show
```

Comment:
Is it the same custom resource as this one: https://kyma-project.io/docs/root/kyma#custom-resource-installation? 
Should it be `installation.installer.kyma-project.io` or `installations.installer.kyma-project.io`?